media/cdm/library_cdm/clear_key_cdm/BUILD.gn

```diff
@@ -46,6 +46,11 @@ loadable_module("clear_key_cdm_bin") {
     ]
     defines += [ "CLEAR_KEY_CDM_USE_FFMPEG_DECODER" ]
     deps += [ "//third_party/ffmpeg" ]
+  } else {
```

Comment:
This file and `remoting/BUILD.gn` change seem like a clean upstream fix. Please send those, if applicable in trunk.